packages/vue/src/radio-button.ts

```diff
@@ -0,0 +1,58 @@
+import RadioButton, { Properties } from "../../jquery/lib/esm/radio-button";
+import { createComponent } from "devextreme-vue/core/index";
+
+type AccessibleOptions = Pick<Properties,
```

Comment:
Minor: The type name makes me think that it is somehow related to accessibility (a11y), which can be a bit confusing.

Reply:
I agree that the naming is a bit confusing.
This file was generated by that tool, we can modify it, but later.